src/documents/FrameworkConcepts.md

```diff
@@ -8,31 +8,56 @@ The Fluid Infusion Framework is built on a number of important concepts. This pa
+We recommend that all objects stored in the `model` area of a component conform to an even stricter definition - that they are equivalent to their form as serialised as JSON. This implies that they
+consist of only the following types: `Array`, `Object`, `String`, `Number` and `Boolean`. In addition, the following values for `Number` should not be used - `Infinity`, `-Infinity` and `NaN`. The value
+`undefined` also can not be safely stored within a model.
 
 An analogy from another environment, Java, is a [POJO](http://en.wikipedia.org/wiki/POJO) or "Plain Old Java Object". The J could just as easily be substituted for "JavaScript" in this case.
 
 ## EL Paths
 
-In some parts of the framework, we refer to "EL expressions". This is a somewhat historical phrase that perhaps sounds like it says more than it is trying to. All we mean by EL expressions are dot-separated paths built of names—for example if you had defined an object `zar` with a member `boo` which has a member `baz`, you could access the nested JavaScript property by writing the expression `zar.boo.baz`. If it were held in a string value, this would become an EL expression—that is, the string `"zar.boo.baz"` is an EL expression which designates the same piece of data we just referenced. The framework includes machinery for interpreting such expressions held in strings rather than at the JavaScript language level. This is useful because it abstracts references to pieces of data from the actual data itself - and to even store these references in documents separately from a running program. It is possible to replace one object tree with another, but still to maintain a stable reference to the same subproperty `baz`, whoever it happens to be today. This is particularly important in web applications where data claiming to be "your data" can suddenly arrive from anywhere (a JSON feed, some persistence, a particularly aggressive version management system, etc). However it got here, you know it is your data because it is at the right path.
+In some parts of the framework, we refer to "EL expressions". This is a somewhat historical phrase that perhaps sounds like it says more than it is trying to. 
+All we mean by EL expressions are dot-separated paths built from names &emdash; for example if you had defined an object `zar` with a member `boo` which has a member `baz`, 
```

Comment:
Oh yes - I had forgotten here that markdown is such a marvellous technology that I need to encode em dashes using the Unicode reference &#8212; ....